app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java

```diff
@@ -206,6 +208,7 @@ public BgGraphBuilder(Context context, long start, long end, int numValues, bool
             loaded_start=start;
             loaded_end=end;
             bgReadings = BgReading.latestForGraph(numValues, start, end);
+            Libre2RawValues = Libre2RawValue.latestForGraph(numValues * 5, start, end);
```

Comment:
Does this make a database query even when not using Libre?

Reply:
You are right. I'll correct that this evening.